source/reporting/index.html.md.erb

```diff
@@ -0,0 +1,113 @@
+---
+title: Reporting 
+weight: 70
+---
+
+# Reporting 
+
+You can extract reporting information from GOV.UK Pay manually, or via the
+API.
+
+## Manual reporting
+
+You can sign in to your [GOV.UK Pay admin
+account](https://selfservice.payments.service.gov.uk/login) and export
+transaction information in a CSV file. 
+
+For a given account, select __Transactions__ in the __Dashboard__ page.
+
+You can filter transactions using the available search criteria, and then
+select __Download all transaction details as a CSV file__. 
+
+You can process CSV files using spreadsheet software.
+
+## Reporting via the GOV.UK Pay API 
+
+Before reading this section, you should read the section describing [Making
+payments](/making_payments). 
+
+You can use the GOV.UK Pay API to export data automatically into your finance
+systems. This requires you to do some development, but could remove 
+routine manual operations from your workflow and lead to significant cost
+savings for your organisation. Before you do this, you should have a datastore 
+where you are already recording payments. 
+
+You can use the GOV.UK Pay API to:
+
+* get information about single payments
+* generate a list of payments 
+
+## Get information about single payments
+
+### Check the status of a single payment
+
+`GET /v1/payments/paymentId`
+
+1. Sign in to <a href="https://gds-payments.gelato.io/api-explorer/"
+   target="blank">the API Explorer</a> (link opens in new window).  
+2. Under __Resource__, select __{Payment Id}. Under __Action__, select <a
+   href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/general/"
+   target="blank">__Find payment by ID__</a> (link opens in new window). 
+   
+For a valid request, the JSON response body contains a `"state"` field. 
+
+For example, for a successful payment:
+
+```
+  "state": {
+    "status": "success",
+    "finished": true
+    },
+```
+
+### See information users have entered for a single payment
```

Comment:
Not sure if this should be in a separate section to the stuff about status. It’s the same API call but the separation makes it look like two different things.

Reply:
Yeah, realised that when I was writing it: I've changed it 👍